### PR TITLE
Don't abort on git.Blame errors

### LINKED ISSUE #966: Blame: error kills connection and stop query.
Executing the following query:

`select commit_hash,explode(blame(repository_id, commit_hash)) from commits limit 100000`

I got this error:

```
INFO[0003] audit trail                                   action=authentication address="127.0.0.1:60820" success=true system=audit user=root
DEBU[0003] executing query                               query="select commit_hash,explode(blame(repository_id, commit_hash)) from commits limit 100000"
INFO[0003] audit trail                                   action=authorization address="127.0.0.1:60820" connection_id=1 permission=read pid=1 query="select commit_hash,explode(blame(repository_id, commit_hash)) from commits limit 100000" success=true system=audit user=root
INFO[0007] audit trail                                   action=query address="127.0.0.1:60820" connection_id=1 duration=3.690559548s err="contents and commits have different length" pid=1 query="select commit_hash,explode(blame(repository_id, commit_hash)) from commits limit 100000" success=false system=audit user=root
ERRO[0007] Error in the middle of a stream to client 1 (127.0.0.1:60820): contents and commits have different length 
INFO[0007] kill query: pid 1                            
INFO[0007] ConnectionClosed: client 1              
```
This error shouldn't stop te query, just add a Warning to check later on SHOW WARNINGS.

### PR DESCRIPTION
**Pull request by juanjux**
Fixes #966 

Signed-off-by: Juanjo Alvarez <juanjo@sourced.tech>

 
 
 - [x] I updated the documentation explaining the new behavior if any.
 - [X] I updated CHANGELOG.md file adding the new feature or bug fix.
 - [x] I updated go-mysql-server using `make upgrade` command if applicable.
 - [x] I added or updated examples if applicable.
 - [x] I checked that changes on schema are reflected into the documentation, if applicable.

### CODE REVIEW

File: internal/function/blame.go [hunk 1]
```diff
@@ -32,7 +36,11 @@ func (g *BlameGenerator) loadNewFile() error {
 
 	result, err := git.Blame(g.commit, g.curFile.Name)
 	if err != nil {
-		return err
+		msg := fmt.Sprintf("Error in BLAME for file %s: %s",
```
Comment (erizocosmico) [inline internal/function/blame.go]: of one of the parameters is in a newline, then better put each in a new line:

```go
msg := fmt.Sprintf(
    "Error in BLAME for file %s: %s",
    g.curFile.Name,
    err.Error(),
)

File: internal/function/blame.go [hunk 2]
```diff
@@ -145,7 +153,7 @@ func (b *Blame) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
 		return nil, err
 	}
 
-	bg, err := NewBlameGenerator(commit, fIter)
+	bg, err := NewBlameGenerator(commit, fIter, ctx)
```
Comment (erizocosmico) [inline internal/function/blame.go]: Context is by convention always the first parameter if a function takes a context.

File: internal/function/blame.go (hunk 1 above)
Comment (juanjux) [reply internal/function/blame.go]: Changed.

File: internal/function/blame.go (hunk 2 above)
Comment (juanjux) [reply internal/function/blame.go]: Fixed, thanks, learnt something new.

**Pull request merged**